testsuite/tests/iterators/int_iterator/main.adb

```diff
@@ -0,0 +1,33 @@
+with Ada.Text_IO; use Ada.Text_IO;
+
+with Libfoolang.Analysis; use Libfoolang.Analysis;
+
+procedure Main is
+   Ctx : constant Analysis_Context := Create_Context;
+   U   : constant Analysis_Unit := Ctx.Get_From_Buffer
+     (Filename => "main.txt",
+      Buffer   => "example");
+begin
+   Put_Line ("main.adb: Running...");
+   if U.Has_Diagnostics then
+      raise Program_Error;
+   end if;
+
+   --  Through array
```

Comment:
I suggest to turn these comments into calls to `Put_Line` for `test.out` readability